<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionarUsuario</class>
 <widget class="QDialog" name="gestionarUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="editUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>800</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Nombre y apellidos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="placeholderText">
     <string>Nombre y apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Usuario&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPass">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Contraseña&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPass">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcpetar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBorrar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboOwner">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOwner">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Owners:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOficina">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Oficinas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboOficina">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>800</width>
     <height>280</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboOwner</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>selectOwner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOficina</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>selectOficina()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNuevo</sender>
   <signal>clicked()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>nuevo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModificar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>modificar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBorrar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>borrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAcpetar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>aceptar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>cancelar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>773</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableUser</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>gestionarUsuario</receiver>
   <slot>showLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectOwner()</slot>
  <slot>selectOficina()</slot>
  <slot>nuevo()</slot>
  <slot>modificar()</slot>
  <slot>borrar()</slot>
  <slot>aceptar()</slot>
  <slot>cancelar()</slot>
  <slot>showLabel()</slot>
 </slots>
</ui>
